main.go

```diff
@@ -22,5 +22,7 @@ func main() {
 	helpers.AlertError(err, "There was an error connecting to the database")
 	fmt.Println("Successfully connected to Mongo")
 
+	helpers.SetupIndexes()
```

Comment:
Make sure to add this to testing as well. You can add it to the connect to mongo helper in `testing_helpers`

Reply:
Good call